.eslintrc.js

```diff
@@ -1,51 +1,60 @@
-'use strict';
-
 module.exports = {
-	root: true,
-	parserOptions: {
-		ecmaVersion: 2017,
-		sourceType: 'module'
-	},
-	plugins: [
-		'ember'
-	],
-	extends: [
-		'plugin:ember/recommended',
+  root: true,
+  parser: 'babel-eslint',
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+    ecmaFeatures: {
+      legacyDecorators: true
+    }
+  },
+  plugins: [
+    'ember',
+		'bbva'
+  ],
+  extends: [
+    'eslint:recommended',
+    'plugin:ember/recommended',
 		'eslint-config-bbva'
-	],
-	env: {
-		browser: true
-	},
-	overrides: [{
-		files: [
-			'.huskyrc.js',
-			'.commitlintrc.js',
-			'.eslintrc.js',
-			'.template-lintrc.js',
-			'ember-cli-build.js',
-			'index.js',
-			'testem.js',
-			'blueprints/*/index.js',
-			'config/**/*.js',
-			'tests/dummy/config/**/*.js'
-		],
-		excludedFiles: [
-			'addon/**',
-			'addon-test-support/**',
-			'app/**',
-			'tests/dummy/app/**'
-		],
-		parserOptions: {
-			sourceType: 'script',
-			ecmaVersion: 2015
-		},
-		env: {
-			browser: false,
-			node: true
-		},
-		plugins: ['node'],
-		rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
-			'no-process-env': 0
-		})
-	}]
+  ],
+  env: {
+    browser: true
+  },
+  rules: {
+    'ember/no-jquery': 'error'
+  },
+  overrides: [
+    // node files
+    {
+      files: [
+				'.huskyrc.js',
+				'.commitlintrc.js',
```

Comment:
Do a `npm run lint:js -- --fix` to fix this.

Reply:
Right